assets/css/edd-admin.css

```diff
@@ -2431,6 +2414,57 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div.edd-reports-export {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```

Comment:
I just realized, with my browser zoom set at 125%, this nicely goes to three boxes across on my screen, but at 100%, it's five across. So although the 300px works nicely for me, it may not be the best option for people with better eyesight? 👓

Reply:
Thoughts @spencerfinnell and @ashleyfae?

My screen has 4 columns across and I think it is very readable.  I tend to err towards covering the person with iffy sight.  We could go a little smaller as these do scale well with browser zooming.

Reply:
I really like how it's behaving currently.

Reply:
I agree--it's a disappointment to switch to another branch and visit this screen now.